plugins/modules/vmware_host_facts.py

```diff
@@ -35,6 +35,12 @@
     default: false
     type: bool
     required: false
+  show_datacenter:
+    description:
```

Comment:
We should document when this has been added:

```suggestion
    version_added: 4.2.0
    description:
```